.github/workflows/tox.yml

```diff
@@ -65,8 +65,14 @@ jobs:
         python -m pip freeze --all
     - name: >-
         Initialize tox envs
-      run: |
-        python -m tox --parallel auto --parallel-live --notest -vv
+      run: >-
+        python -m
+        tox
+        --parallel auto
+        --parallel-live
+        --notest
+        --skip-missing-interpreters false
```

Comment:
Note: this is a technique that tox itself employs in their CI. I guess I forgot to transfer it here earlier.

Reply:
I still consider that we need to set skip-missing-interpreters=False inside tox.ini but I do not want to delay this even further as nobody else cared to review it. I will merge as it but I hope you will support a change on tox.ini when zuul will fail to use right python version.

Reply:
I cannot support it without seeing any evidence